mne/channels/tests/test_montage.py

```diff
@@ -555,6 +555,13 @@ def test_set_montage():
     r0 = _fit_sphere(new_pos)[1]
     assert_allclose(r0, [0., -0.016, 0.], atol=1e-3)
 
+    # test if set_montage works with a string argument
+    raw1 = read_raw_fif(fif_fname)
```

Comment:
can you rather update a test? just to limit testing time.

Reply:
OK, I'll just replace it in this function then.

Reply:
Done.